views/default/css/admin.php

```diff
@@ -1300,42 +1300,43 @@
 /* ***************************************
 	ICONS
 *************************************** */
+.fa {
+	color: #aaa;
+	font-size: 16px;
+}
+
+:focus > .elgg-icon,
+.elgg-icon:hover,
+.elgg-icon-hover {
+	color: #333;
+}
+
+h1 > .fa,
+h2 > .fa,
+h3 > .fa,
+h4 > .fa,
+h5 > .fa,
+h6 > .fa {
+	font-size: 1em;
```

Comment:
in Elgg 1em = 14.4px because of https://github.com/Elgg/Elgg/blob/master/views/default/elements/typography.css.php#L15